WordPress/src/test/java/org/wordpress/android/ui/posts/prepublishing/PrepublishingTagsViewModelTest.kt

```diff
@@ -32,7 +32,7 @@ class PrepublishingTagsViewModelTest : BaseUnitTest() {
     @Test
     fun `when viewModel is started updateToolbarTitle is called with the tags title`() {
         var title: UiStringRes? = null
-        viewModel.updateToolbarTitle.observeForever {
```

Comment:
This change was supposed to be made in a previous PR. Corrected the name of this here so the build passes.